src/endpoint/s3/ops/s3_put_bucket_cors.js

```diff
@@ -13,7 +13,7 @@ async function put_bucket_cors(req) {
             allowed_methods: rule.AllowedMethod,
             allowed_origins: rule.AllowedOrigin,
             expose_headers: rule.ExposeHeader,
-            id: rule.ID,
+            id: rule.ID?.[0],
```

Comment:
Not sure I understand why it is an array.
Looking at https://docs.aws.amazon.com/AmazonS3/latest/API/API_CORSRule.html it should be a sting.
will it always be array? 
we can make sure by doing:
```suggestion
            id: (typeof rule.ID === "string") ? rule.ID : rule.ID?.[0],
```
WDYT?

Reply:
I agree about the documentation, that's what confused us in the 1st place. This was tested with AWS cli and with node.js SDK (Added also a test) and as they can do all sorts of future changes we will support only this format for now.